src/SFA.DAS.LevyTransferMatching.Web/Validators/Opportunities/ContactDetailsPostRequestValidator.cs

```diff
@@ -0,0 +1,53 @@
+﻿using FluentValidation;
+using SFA.DAS.LevyTransferMatching.Web.Models.Opportunities;
+using System.Net.Mail;
+
+namespace SFA.DAS.LevyTransferMatching.Web.Validators.Opportunities
+{
+    public class ContactDetailsPostRequestValidator : AbstractValidator<ContactDetailsPostRequest>
+    {
+        private const string EmailAddressError = "Enter your email address";
+
+        public ContactDetailsPostRequestValidator()
+        {
+            RuleFor(x => x.FirstName)
+                .NotEmpty()
+                .WithMessage("Enter your first name");
+
+            RuleFor(x => x.LastName)
+                .NotEmpty()
+                .WithMessage("Enter your last name");
+
+            RuleFor(x => x.EmailAddress)
+                .NotEmpty()
+                .WithMessage(EmailAddressError);
+
+            When(
+                x => !string.IsNullOrEmpty(x.EmailAddress),
+                () =>
+                {
+                    RuleFor(x => x.EmailAddress)
+                        .Must(ValidEmailAddress)
+                        .WithMessage(EmailAddressError);
+                });
+
+            RuleForEach(x => x.AdditionalEmailAddresses)
+                .Must(x => string.IsNullOrEmpty(x) || (!string.IsNullOrEmpty(x) && ValidEmailAddress(x)))
+                .WithMessage(EmailAddressError);
```

Comment:
"Enter your email address" doesn't seem like an ideal error message when a user has entered something that is not valid, but I'm unsure what was asked for so will leave it with you to check.

Reply:
Hi, @chrisfoster76 - I have checked on this one with Sam Wing - he confirmed that I use the same error message. I will forward you the conversation on Slack.